tests/detect-bypass/test.yaml

```diff
@@ -0,0 +1,22 @@
+args:
+- --set stream.midstream=true
+
+checks:
+- filter:
+    count: 1
+    match:
+      event_type: alert
+      alert.signature_id: 1
+- filter:
+    count: 1
```

Comment:
The CI failures indicate that we are not getting this match here, so do check what's going on, please! 🤔

Reply:
I think this is also due the flow directions.. should be fine once that is fixed.